test/server/https-option.test.js

```diff
@@ -42,9 +42,7 @@ describe('https option', () => {
       req.get('/').expect(200, /Heyo/, done);
     });
 
-    afterAll(() => {
-      testServer.close();
-    });
```

Comment:
`testServer.close` requires a callback, causing `TypeError: done is not a function`.